### ISSUE TITLE
Too many component groups!

### ISSUE THREAD

**Issue by ylep** [brainvisa/brainvisa-cmake#72]
@denisri I see that you introduced the `cea` and `bvdev` component groups today, this is good for clarity but now the profusion of groups make it difficult to understand:

- `all` is not very useful anymore (it is basically `cea` + obsolete/internal projects, but people who want to work on the obsolete/internal stuff can add it to their `bv_maker.cfg`)
- The differences between `cea`, `bvdev`, and `standard` are subtle... I still do not understand why we need all 3 of those

**Comment by denisri** [comment]
Yes I did myself more or less the same remark as I was doing it... I did not remove anything for now because people are already using groups that we may want to remove. 
To clarify, currently:
* "brainvisa" and "cea" contain the projects we actually release in binary distributions, respectively in the public distrib on the web site, and installed in neurospin/mircen/shfj for "internal" users
* "bvdev" and "standard" are more or less the same as the above ones (respectively), but for our developers who need to retrieve the sources, with an account on the bioproj server with "standard" access permissions - which means that a few project(s) are removed compared to "brainvisa" and "cea" because they have very restrictive access permissions to sources which most developers do not have (but they are released as binary versions in the official distribs).

But in the future, maybe:
* "all" is not needed, we could use "cea" plus (if needed) a few additional projects added by hand. Anyway "all" is already not all the projects in our repositories, so its name is not accurate any longer.
* "standard" would become useless
* brainvisa and bvdev would become the same again (depends if brainrat-private could open its sources at the same level as other projects)
* opensource and brainvisa could also merge, when we actually make our "private" projects public (which has been decided several years ago but we still haven't done the technical operations to actually make them public).
